src/overlays/actors/ovl_En_Dnb/z_en_dnb.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_en_dnb.c
+ * Overlay: ovl_En_Dnb
+ * Description: Dnb
```

Comment:
```suggestion
 * Description: 
```
Better to just leave this blank if you don't know IMO.

Reply:
Agreed.